src/main/resources/database-setup/patches/MARCCAT_DB/1.6/MODCAT-210.sql

```diff
@@ -0,0 +1,1142 @@
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '1', ' ', 'NH', 11, 3, 7, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '1', ' ', 'SH', 4, 9, 2, -1, 'SU');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '1', ' ', 'TH', 11, 3, 7, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '2', ' ', 'MH', 2, 3, 8, 2, 'NA');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '2', ' ', 'MH', 11, 3, 8, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '2', ' ', 'NH', 2, 3, 8, 2, 'NA');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '2', ' ', 'NH', 11, 3, 8, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '2', ' ', 'SH', 4, 10, 2, -1, 'SU');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '2', ' ', 'TH', 11, 3, 8, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '@', ' ', 'MH', 2, 3, 1, 2, 'NA');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '@', ' ', 'MH', 11, 3, 1, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '@', ' ', 'NH', 2, 3, 1, 2, 'NA');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '@', ' ', 'NH', 11, 3, 1, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('410', '@', ' ', 'TH', 11, 3, 1, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '0', ' ', 'MH', 2, 4, 6, 2, 'NA');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '0', ' ', 'MH', 11, 4, 6, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '0', ' ', 'NH', 2, 4, 6, 2, 'NA');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '0', ' ', 'NH', 11, 4, 6, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '0', ' ', 'SH', 4, 12, 2, -1, 'SU');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '0', ' ', 'TH', 11, 4, 6, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '1', ' ', 'MH', 2, 4, 7, 2, 'NA');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '1', ' ', 'MH', 11, 4, 7, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '1', ' ', 'NH', 2, 4, 7, 2, 'NA');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '1', ' ', 'NH', 11, 4, 7, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '1', ' ', 'SH', 4, 13, 2, -1, 'SU');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '1', ' ', 'TH', 11, 4, 7, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '2', ' ', 'MH', 2, 4, 8, 2, 'NA');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '2', ' ', 'MH', 11, 4, 8, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '2', ' ', 'NH', 2, 4, 8, 2, 'NA');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '2', ' ', 'NH', 11, 4, 8, 2, ' ');
+Insert into OLISUITE.S_AUT_MARC_IND_DB_CRLTN VALUES ('411', '2', ' ', 'SH', 4, 14, 2, -1, 'SU');
+COMMIT;
```

Comment:
Just curious why the commit is here vs. the end of the file?

Reply:
I have added at the end of the file